gamemodes/terrortown/gamemode/shared/lang/polish.lua

```diff
@@ -1,1223 +1,1223 @@
-L.f1_settings_hudswitcher_desc_layout_editor = "Zmień położenie i\nrozmiar elementów"
-L.f1_settings_hudswitcher_button_layout_editor = "Edytor położenia"
-L.f1_settings_hudswitcher_desc_hud_not_supported = "! TEN HUD NIE WSPIERA EDYCJI!"
-
-L.f1_bind_reset_default = "Podstawowy"
-L.f1_bind_disable_bind = "Wyczyść"
-L.f1_bind_description = "Kliknij i naciśnij guzik, aby go zbindować."
-L.f1_bind_reset_default_description = "Resetuj do podstawowego guzika."
-L.f1_bind_disable_description = "Wyczyść bind."
-L.ttt2_bindings_new = "Nowy klawisz dla {name}: {key}"
-
-L.f1_bind_weaponswitch = "Zmień broń"
-L.f1_bind_sprint = "Sprint"
-L.f1_bind_voice = "Globalny Czat"
-L.f1_bind_voice_team = "Teamowy Czat"
-
-L.f1_dmgindicator_title = "Ustawienia powiadomień obrażeń"
-L.f1_dmgindicator_enable = "Zezwól"
-L.f1_dmgindicator_mode = "Wybierz motyw"
-L.f1_dmgindicator_duration = "Liczba sekund widoczności"
-L.f1_dmgindicator_maxdamage = "Obrażenia do maksymalnej widoczności"
-L.f1_dmgindicator_maxalpha = "Obrażenia do minimalnej widoczności"
-
-L.ttt2_bindings_new = "Nowy klawisz dla {name}: {key}"
-L.hud_default = "Zwykły HUD"
-L.hud_force = "Wymuszony HUD"
-L.hud_restricted = "Zakazany HUD"
-L.hud_default_failed = "Failed to set the HUD {hudname} as new default. You don't have permission to do that, or this HUD doesn't exist."
-L.hud_forced_failed = "Failed to force the HUD {hudname}. You don't have permission to do that, or this HUD doesn't exist."
-L.hud_restricted_failed = "Failed to restrict the HUD {hudname}. You don't have permission to do that."
-
-L.shop_role_select = "Wybierz rolę"
-L.shop_role_selected = "{roles} wybrano do sklepu!"
-L.shop_search = "Szukaj"
-
-L.button_save = "Zapisz"
-
-L.disable_spectatorsoutline = "Wyłącz otoczki przedmiotów"
-L.disable_spectatorsoutline_tip = "Więcej fps"
-
-L.disable_overheadicons = "Wyłącz Ilony nad głową"
-L.disable_overheadicons_tip = "Więcej fps"
-
--- 2020-01-04
-L.doubletap_sprint_anykey = "Continue double tap sprinting until you stop moving"
-L.doubletap_sprint_anykey_tip = "You will keep sprinting as long as you keep moving"
-
-L.disable_doubletap_sprint = "Wyłącz sprint poprzez wielokrotne wciskanie"
-L.disable_doubletap_sprint_tip = "Double tapping a movement key will no longer cause you to sprint"
-
--- 2020-02-03
-L.hold_aim = "Trzymaj aby celować"
-L.hold_aim_tip = "Będzie działać tak długo jak będziesz trzymać klawisz"
-
--- 2020-02-09
-L.name_door = "Drzwi"
-L.door_open = "Naciśnij [{usekey}] aby otworzyć drzwi."
-L.door_close = "Naciśnij [{usekey}] aby zamknąć drzwi"
-L.door_locked = "Drzwi są zamknięte"
-
--- 2020-02-11
-L.automoved_to_spec = "(SERWER) Zostałeś przeniesiony do teamu obserwujących za brak aktywności!"
-
--- 2020-02-16
-L.door_auto_closes = "Te drzwi zamykają się same"
-L.door_open_touch = "Wejdź w drzwi żeby je otworzyć."
-L.door_open_touch_and_use = "Wejdź w drzwi i naciśnij [{usekey}] aby otworzyć."
-L.hud_health = "Zdrowie"
+-- Polish language strings
```

Comment:
What happened with this file?

Reply:
Lebroomer suspected that they used different end of lines